docs/FAQ/FAQ-General.md

```diff
@@ -46,129 +44,50 @@ BTCPay doesn't compromise on censorship-resistance, which is one of the main fea
 
 BTCPay server is a feature-rich software with plenty of use-cases that can solve problems for different types of users. Merchants, content creators, lightning network users, exchanges, hosting providers and many others can find it useful. See the [Use Case page](../UseCase.md) for detailed breakdown of BTCPay use-cases.
 
-## How much does it cost to run BTCPay Server?
-
-BTCPay is completely free software. The software itself costs nothing. It can be downloaded and used by anyone, free of charge. You can run it **internally** as a self-hosted solution, on a home server, on a low-cost single-board computer like [Raspberry Pi](../RaspberryPiDeployment.md) or different kind of [in-home hardware](../HardwareDeployment.md)
-
-Hosting BTCPay **externally**, usually means using a Virtual Private Server, enterprise-grade infrastructure, or a [third-party BTCPay hosting](../ThirdPartyHosting.md).
-
-The price of hosting BTCPay Server obviously depends on the hosting provider.
-
-The low-priced VPS servers tested by the community start at around [9$/month](../LunaNodeWebDeployment.md), but you're free to explore other hosting providers that suit your budget, as long as they meet the [minimum requirements](FAQ-General.md#what-are-the-minimal-requirements-to-run-btcpay).
-
-### Why do I need a VPS? Can't I just run BTCPay on my home computer?
-
-Just like you can host a website locally, you can do it with BTCPay. Please be aware that your PC would have to meet the minimal requirements and also run 24/7 if you don't want interruptions of service.
-
-You might also not want to expose your home IP address for an activity related to payment and money.
-
-For all these reasons, while local hosting is suitable for testing, it's not a viable solution for production.
-
-### Can the cost of running BTCPay on Microsoft Azure be reduced?
-
-Please check out [How to penny-pinch your Azure deployment](../AzurePennyPinching.md).
-
-## What are the minimal requirements to run BTCPay?
-
-If you would like to run Bitcoin and Lightning Network nodes, the minimal requirements are:
-
-- 2GB Ram
-- 80 GB of storage (with [pruning](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md#generated-docker-compose) enabled)
-- Docker
+BTCPay is licensed under the [MIT License](https://github.com/btcpayserver/btcpayserver/blob/master/LICENSE). 
 
 ## How to install BTCPay Server?
 
-If you don't have the technical knowledge, we recommend that you use our new web-wizard to deploy BTCPay Server.
-
-- [How to install BTCPay via Web-Wizard LunaNode - guide](https://medium.com/@BtcpayServer/launch-btcpay-server-via-web-interface-and-deploy-full-bitcoin-node-lnd-in-less-than-a-minute-dc8bc6f06a3)
-- [How to install BTCPay via Web-Wizard LunaNode - video](https://www.youtube.com/watch?v=NjslXYvp8bk)
+First take a look at the various deployment options and consider which one best suits your specific needs.
```

Comment:
Perhaps a link in `deployment options`?

Reply:
There is a link to deployments page below that line. Deployments FAQ has a which is the easiest way to install. And Deployments.md takes you to LunaNodeWebDeployment.md which has links to both the LunaNode medium article and video. 

Is one of those what you are saying or are you not talking about a lunanode link?